packages/sygma-ui/src/contexts/NetworkManagerContext/NetworkManagerContext.tsx

```diff
@@ -44,11 +50,15 @@ interface NetworkManagerContextInterface {
 
   domainId?: number;
 
-  homeChainConfig: BridgeConfig | undefined;
-  destinationChainConfig: BridgeConfig | undefined;
+  homeChainConfig: BridgeConfig | EvmBridgeConfig | SubstrateConfig | undefined;
```

Comment:
I wonder here, are we implementing the interfaces or are we using those as types? If so, shouldn't we convert this interface to a type?

Reply:
I don't have strong opinion on this. The contexts are kind sophisticated ones. It looks like it could be both

Reply:
lets leave like this in the meantime